src/openseadragon.js

```diff
@@ -1209,6 +1223,12 @@ function OpenSeadragon( options ){
                     HOVER:  'rotateright_hover.png',
                     DOWN:   'rotateright_pressed.png'
                 },
+                flip: { // Flip icon by Yaroslav Samoylov from the Noun Project
```

Comment:
I figure this is a fine spot to include the credit, but you might as well include the URL.